sdks/go/pkg/beam/core/runtime/exec/datasource.go

```diff
@@ -189,6 +202,23 @@ func (n *DataSource) String() string {
 	return fmt.Sprintf("DataSource[%v] Coder:%v Out:%v", n.SID, n.Coder, n.Out.ID())
 }
 
+// IncrementCountAndCheckSplit increments DataSource.count by one and checks to
+// make sure the new value is smaller than the promised split point. If the new
+// value is greater than or equal to the split point, calls the FinishBundle and
```

Comment:
This code no longer calls FinishBundle directly, so we should remove that part of the comment.

What do you think of this alternative documentation? :

IncrementCountAndCheckSplit increments DataSource.count by one and checks if the caller should abort further element processing, and finish the bundle.
Returns true if the new value of count is greater than or equal to the split point, and false otherwise.

Reply:
done.